bin/delete_bioentities_species.sh

```diff
@@ -6,13 +6,13 @@ source $scriptDir/common_routines.sh
 require_env_var "SOLR_HOST"
 require_env_var "SPECIES"
 
-HTTP_STATUS=$(curl -o >(cat >&3) -w "%{http_code}" -X POST -H 'Content-Type: application/json' \
-"http://$SOLR_HOST/solr/bioentities-v1/update/json?commit=true" --data-binary \
-'{
-  "delete": {
-    "query": "species:$SPECIES"
```

Comment:
I guess we can keep JSON with:
```
    "query": "'species:$SPECIES'"
```

Reply:
as long as $SPECIES is within single quotes it won't work, it won't get evaluated.